### PR TITLE
Add proposal for Multiple clusters configuration

### PR DESCRIPTION
**Pull request by yinshiua**
Adds a proposal document for multiple clusters configuration.

Related to #59, #71, #112, #152, #174, #176

Related to proposal https://github.com/aws/amazon-ecs-cli/pull/252

### CODE REVIEW

File: proposals/multi-cluster-config.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+Does not need to configure cluster
+1. Configure ECS profile
+   ```
+   ecs-cli configure  profile --access-key ********** --secret-key ********** --profile-name my_profile
+   ```
+1. Cluster PS
+   ```
+   ecs-cli ps --cluster cli-demo --region us-west-2 --ecs-profile my_profile
+   ```
+1. Do the same with AWS profile
```
Comment (markriggins) [inline proposals/multi-cluster-config.md]: 👍 -- that's what I really need for use on Jenkins, when we really don't want to contend with multiple projects that need to work with various clusters simultaneously

File: proposals/multi-cluster-config.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+# Workflow
+### Single cluster first run
+1. Configure ECS credentials (Stores in `~/.ecs/ecs.profile`)
+   ```
+   ecs-cli configure  profile --access-key ********** --secret-key ********** --profile-name my_profile
+   ```
+1. Configure cluster (Stores in `~/.ecs/ecs.config`)
+   ```
+   ecs-cli configure --cluster cli-demo --region us-west-2 --config-name beta_config
+   ```
```
Comment (markriggins) [inline proposals/multi-cluster-config.md]: it's not clear to me how the **config-name** of `beta_config` factors into this.   It does not appear to be a field in the ~/.ecs/ecs.config YAML.   Does this result in a  new`~/.ecs/beta_config` file, or a section in the ~/.ecs/ecs.config file?

Comment (yinshiua) [inline proposals/multi-cluster-config.md]: We allow multiple cluster configurations in ~/.ecs/ecs.config, `--config-name` is a way to identify the different cluster profile which can be specified at runtime. Assume you have an existing cluster config "ecs" and you would like to configure a different cluster but not overwriting the previous one, you can run the command with `--config-name` with "beta_config" to add the new cluster in the cluster config.
```yaml
default: ecs
clusters:
  ecs: 
      cluster: ecs-test
      region: us-west-1
  beta_config:
      cluster: cli-demo
      region: us-west-2
```

Comment (markriggins) [inline proposals/multi-cluster-config.md]: if these are "configurations" why are we listing them under "clusters:" in the above YAML?

how about simply doing this instead?
```
default: ecs-test
clusters:
   ecs:
      region: use-west-1
   cli-demo:
      region: us-west-2
```

File: proposals/multi-cluster-config.md (hunk 1 above)
Comment (gstaples) [inline proposals/multi-cluster-config.md]: Isn't it redundant to specify --region when it's already in the profile?  Why not just use --profile like every other AWS command?

Comment (yinshiua) [inline proposals/multi-cluster-config.md]: --region is a way for you to overwrite your cluster configuration which comes in handy when you are running script to perform the same actions in multiple regions.

**Pull request merged**